src/routes/instance.rs

```diff
@@ -189,13 +191,41 @@ fn ban(_admin: Admin, conn: DbConn, id: i32) -> Redirect {
     Redirect::to(uri!(admin_users))
 }
 
+struct Headers<'r> {
+    headers: HeaderMap<'r>,
+}
```

Comment:
This should probably go somewhere else, as it will be used on both the shared inbox and the personal one, which is defined in another file